.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Not sure why you committed these editor files. It seems you should add `.idea/` to the `.gitignore` file and run:

```bash
git rm -r -f .idea
```

to remove these files from the repository.